sycl/include/CL/sycl/detail/pi.h

```diff
@@ -88,6 +88,65 @@ typedef enum {
   PI_QUEUE_INFO_REFERENCE_COUNT = CL_QUEUE_REFERENCE_COUNT
 } _pi_queue_info;
 
+// TODO: populate
+typedef enum {
+  PI_IMAGE_FORMAT       = CL_IMAGE_FORMAT,
+  PI_IMAGE_ELEMENT_SIZE = CL_IMAGE_ELEMENT_SIZE,
+  PI_IMAGE_ROW_PITCH    = CL_IMAGE_ROW_PITCH,
+  PI_IMAGE_SLICE_PITCH  = CL_IMAGE_SLICE_PITCH,
+  PI_IMAGE_WIDTH        = CL_IMAGE_WIDTH,
+  PI_IMAGE_HEIGHT       = CL_IMAGE_HEIGHT,
+  PI_IMAGE_DEPTH        = CL_IMAGE_DEPTH
+} _pi_image_info;
+
+// TODO: populate
+typedef enum {
+  PI_MEM_OBJECT_BUFFER         = CL_MEM_OBJECT_BUFFER,
+  PI_MEM_OBJECT_IMAGE2D        = CL_MEM_OBJECT_IMAGE2D,
+  PI_MEM_OBJECT_IMAGE3D        = CL_MEM_OBJECT_IMAGE3D,
+  PI_MEM_OBJECT_IMAGE2D_ARRAY  = CL_MEM_OBJECT_IMAGE2D_ARRAY,
+  PI_MEM_OBJECT_IMAGE1D        = CL_MEM_OBJECT_IMAGE1D,
+  PI_MEM_OBJECT_IMAGE1D_ARRAY  = CL_MEM_OBJECT_IMAGE1D_ARRAY,
+  PI_MEM_OBJECT_IMAGE1D_BUFFER = CL_MEM_OBJECT_IMAGE1D_BUFFER
+} _pi_mem_object_type;
+
+// TODO: populate
```

Comment:
could this "populate" be more specific? What is missing so other people can contribute to it?

Reply:
Removed TODOs as it's populated enough to support things SYCL needs.